src/components/Card/Card.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import styled from 'styled-components';
+import { pxToRem, ComponentBaseProps } from '../../shared';
+import { Box, BoxVariant } from '../Box';
+
+/**
+ * Dimensions of card component
+ */
+const cardDimensions = {
```

Comment:
Jotta olisi yhdenmukaista, niin näissä saisi olla pelkkiä lukuja. pxToRem ympärille sitten siellä missä näitä käytetään

Reply:
korjattu